File: 104/104.c
/**
 * Definition for a binary tree node.
 * struct TreeNode {
 *     int val;
 *     struct TreeNode *left;
 *     struct TreeNode *right;
 * };
 */
int max(int a, int b) {
    return a > b ? a:b;
}
int maxDepth(struct TreeNode* root) {
    return root == NULL ? 0 : max(maxDepth(root->left),maxDepth(root->right))+1;
}

File: 104/104.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, x):
#         self.val = x
#         self.left = None
#         self.right = None

class Solution:
    # @param {TreeNode} root
    # @return {integer}
    def max(self, a, b):
        return a if a > b else b
    def maxDepth(self, root):
        return 0 if not root else max(self.maxDepth(root.left),self.maxDepth(root.right))+1;
